packages/service/lib/schema.js

```diff
@@ -118,6 +118,81 @@ const server = {
           additionalProperties: true
         }
       ]
+    },
+    ignoreTrailingSlash: {
+      type: 'boolean'
+    },
+    ignoreDuplicateSlashes: {
+      type: 'boolean'
+    },
+    connectionTimeout: {
+      type: 'integer'
+    },
+    keepAliveTimeout: {
+      type: 'integer'
+    },
+    maxRequestsPerSocket: {
+      type: 'integer'
+    },
+    forceCloseConnections: {
+      anyOf: [
+        { type: 'boolean' },
+        { type: 'string', pattern: '^idle$' }
+      ]
+    },
+    requestTimeout: {
+      type: 'integer'
+    },
+    bodyLimit: {
+      type: 'integer'
+    },
+    maxParamLength: {
+      type: 'integer'
+    },
+    disableRequestLogging: {
+      type: 'boolean'
+    },
+    exposeHeadRoutes: {
+      type: 'boolean'
+    },
+    onProtoPoisoning: {
+
+    },
+    onConstructorPoisoning: {
+
+    },
```

Comment:
I would not allow the user to set `onProtoPoisoning` and `onConstructorPoisoning`